spec/factories/result_table.rb

```diff
@@ -3,5 +3,6 @@
 FactoryBot.define do
   factory :result_table do
     table { create(:table) }
+    result { create(:result) }
```

Comment:
of `result` is association with model named `result` you don't need `{ ... }`

Reply:
:+1: